eval/hf_runner.py

```diff
@@ -115,12 +132,15 @@ def run_hf_eval(
     with tqdm(total=len(df)) as pbar:
         for row in df.to_dict("records"):
             total_tried += 1
+            gc.collect()
+            torch.cuda.empty_cache()
+            torch.cuda.synchronize()
```

Comment:
somehow the code worked (no OOM) only after I added these 3 lines before the generation. not deep diving here - will leave it as is until we figure out the root cause

Reply:
Nice!